src/DependencyInjection/Admin/TaggedAdminInterface.php

```diff
@@ -91,6 +91,11 @@ public function getLabel();
      */
 //    public function showMosaicButton(bool $isShown): void;
 
+    /**
+     * @return array<string, array<string, mixed>>
+     */
+    public function getListModes();
```

Comment:
Keep in mind, that this is a BC break. 

The new interface is not released, so this is okay for now.

Reply:
Yeah, I take advantage of the fact it's not released :)